internal/commands/login/command.go

```diff
@@ -76,6 +76,16 @@ func (cmd *Command) Flags() []flags.Flag {
 				},
 			},
 		},
+		flags.BoolFlag{
+			Value: &cmd.inputs.NoBrowser,
+			Meta: flags.Meta{
+				Name:   "no-browser",
+				Hidden: true,
+				Usage: flags.Usage{
+					Description: "Don't try to open a browser session",
```

Comment:
we should probably (always) get design's feedback on the wording for these ... but I think in general try to avoid contractions: `don't` = `do not`

also I imagine we wait to hear back from sumedha about the flags for this command in general, but are we sure this wants to be hidden?